tests/ops/integration_tests/test_integration_custom_privacy_request_fields.py

```diff
@@ -0,0 +1,177 @@
+                    label="Subscriber IDs", value=["123", "456"]
+                ),
+                "account_ids": CustomPrivacyRequestField(
+                    label="Account IDs", value=[123, 456]
+                ),
+            },
+        }
+
+        mock_response = Response()
+        mock_response.status_code = 200
+        mock_response._content = json.dumps([{"id": 1}]).encode()
+        mock_send.return_value = mock_response
+
+        get_privacy_request_results(
+            db,
+            policy,
+            run_privacy_request_task,
+            data,
+        )
+
+        request_params = mock_send.call_args[0][0]
+        assert request_params.query_params == {"first_name": "John"}
+        assert json.loads(request_params.body) == {
+            "last_name": "Doe",
+            "order_id": None,
+            "subscriber_ids": ["123", "456"],
+            "account_ids": [123, 456],
+        }
+
+    @pytest.mark.usefixtures(
+        "custom_privacy_request_fields_config",
+        "custom_privacy_request_fields_dataset",
+        "allow_custom_privacy_request_field_collection_enabled",
+        "allow_custom_privacy_request_fields_in_request_execution_enabled",
+    )
+    @mock.patch("fides.api.service.connectors.saas_connector.AuthenticatedClient.send")
+    def test_custom_privacy_request_fields_erasure(
+        self, mock_send, db: Session, erasure_policy: Policy, run_privacy_request_task
+    ) -> None:
+        data = {
+            "requested_at": "2021-08-30T16:09:37.359Z",
+            "policy_key": erasure_policy.key,
+            "identity": {"email": "customer@example.com"},
+            "custom_privacy_request_fields": {
+                "first_name": CustomPrivacyRequestField(
+                    label="First name", value="John"
+                ),
+                "last_name": CustomPrivacyRequestField(label="Last name", value="Doe"),
+                "subscriber_ids": CustomPrivacyRequestField(
+                    label="Subscriber IDs", value=["123", "456"]
+                ),
+                "account_ids": CustomPrivacyRequestField(
+                    label="Account IDs", value=[123, 456]
+                ),
+            },
+        }
+
+        mock_response = Response()
+        mock_response.status_code = 200
+        mock_response._content = json.dumps([{"id": 1}]).encode()
+        mock_send.return_value = mock_response
+
+        get_privacy_request_results(
+            db,
+            erasure_policy,
+            run_privacy_request_task,
+            data,
+        )
+
+        request_params = mock_send.call_args[0][0]
+        assert request_params.query_params == {}
+        assert json.loads(request_params.body) == {
+            "user_info": {
+                "first_name": "John",
+                "last_name": "Doe",
+                "subscriber_ids": ["123", "456"],
+                "account_ids": [123, 456],
+            }
+        }
```

Comment:
🏆 Thank you for making this test.  An additional use case for an integer identifier would be good -

Reply:
Use case added!